src/lib/pages/frontsite/Home/Sections/SectionsWrapper.js

```diff
@@ -25,6 +23,7 @@ export default class SectionsWrapper extends Component {
                 .query()
                 .withDocumentType('BOOK')
                 .sortBy('mostrecent')
+                .withSize(5)
```

Comment:
just a small thingy: the `5` should be coming from props, not to be hardcoded, like previously. You could add a prop which has a default equal 5